include/ceed-impl.h

```diff
@@ -280,6 +280,8 @@ struct CeedContextFieldLabel_private {
   size_t                 offset;
   CeedInt                num_sub_labels;
   CeedContextFieldLabel *sub_labels;
+  CeedInt                num_operators;
+  CeedOperator          *op;
```

Comment:
I should perhaps comment here in the code that this is only for checking compatibility and not in any way for calling any op methods